tests/std/tests/P0896R4_views_split/test.cpp

```diff
@@ -153,63 +156,26 @@ constexpr void test_one(Base&& base, Delimiter&& delimiter, Expected&& expected)
 
     // Validate split_view::begin
     STATIC_ASSERT(CanMemberBegin<R>);
-    if (ranges::forward_range<Base>) { // intentionally not if constexpr
-        const auto i = r.begin();
-        if (!is_empty) {
-            assert(ranges::equal(*i, *ranges::begin(expected)));
-        }
-
-        if constexpr (copyable<R>) {
-            auto r2       = r;
-            const auto i2 = r2.begin();
-            if (!is_empty) {
-                assert(ranges::equal(*i2, *i));
-            }
-        }
+    STATIC_ASSERT(!CanMemberBegin<const R>);

```

Comment:
We can strengthen this to "no `const` `begin`, member or otherwise":
```suggestion
    STATIC_ASSERT(!CanBegin<const R&>);
```